nilearn/plotting/img_plotting.py

```diff
@@ -675,12 +885,12 @@ def plot_stat_map(stat_map_img, bg_img=MNI152TEMPLATE, cut_coords=None,
     # dim the background
     bg_img, black_bg, bg_vmin, bg_vmax = _load_anat(bg_img, dim=dim,
                                                     black_bg=black_bg)
-    
+
```

Comment:
For next PR: try not to fix PEP8 errors in random code. Only fix them if you work in the concerned code.